docs/source/tutorial/create_a_pipeline.md

```diff
@@ -149,7 +149,7 @@ You should see output similar to the below:
 You can run the `preprocess_shuttles` node similarly. To test both nodes together as the complete data processing pipeline:
 
 ```bash
-kedro run
```

Comment:
This should include both node names, right?

Reply:
The text accompanying that specific code suggests using it for shuttles next, but feel free to include both if that's what's needed